src/Effects.js

```diff
@@ -743,6 +810,98 @@ module.exports = class Effects {
       }
     };
 
+    this.stars = {
+      star: [],
+      draw: function () {
+        p5.background('black');
+        let star = {
+          x: p5.random(0, 400),
+          y: p5.random(0, 400),
+          size: p5.random(15, 30),
+        };
+        this.star.push(star);
+        p5.noStroke();
+        p5.fill(255, 255, 0);
+        this.star.forEach(function (star){
+          p5.push();
+          p5.translate(star.x, star.y);
+          for (let i = 0; i < 3; i++) {
+            p5.rotate(360 / 5);
+            p5.ellipse(0, 0, 1, star.size);
+          }
+          let fadeSpeed = p5.map(star.size, 15, 30, 1, 2);
+          star.size = star.size - fadeSpeed;
+          star.y = star.y - 2;
+          p5.pop();
+        });
+        this.star = this.star.filter(function (star) {
+          return star.size > 0.1;
+        });
+      }
+    };
+
+    this.starspace = {
+      space: [],
+      draw: function () {
+        p5.background('black');
+        for (let i = 0; i < 3; i ++) {
+          let space = {
+            x: 200,
+            y: 200,
+            velocity: p5.createVector(0, 1).rotate(p5.random(0,360)),
+            size: 0.01,
+          };
+          this.space.push(space);
+        }
+        p5.noStroke();
+        p5.fill('white');
+        this.space.forEach(function (space){
+          p5.push();
+          p5.translate(space.x, space.y);
+          p5.ellipse(0, 0, space.size, space.size);
+          let speedMultiplier = p5.pow(space.size, 2) /2;
+          space.x += space.velocity.x * speedMultiplier;
+          space.y += space.velocity.y * speedMultiplier;
```

Comment:
Sooo... velocity is a constant, and acceleration is controlled by the radius of the ellipse?

Reply:
discussed looking further at this in next version.